src/main/java/org/junit/Assert.java

```diff
@@ -812,6 +812,37 @@ public static void assertNotSame(Object unexpected, Object actual) {
         assertNotSame(null, unexpected, actual);
     }
 
+    /**
+     * Asserts that two {@link CharSequence} instances have the same contentsa.
+     * If they do not, an {@link AssertionError} is thrown.
+     *
+     * @param expected the expected contents
+     * @param actual the object to compare to
+     */
+    public static void assertContentEquals(CharSequence expected, CharSequence actual)  {
```

Comment:
Well I decided to use `CharSequence` as the expected as String implements this. So passing in a literal would be fine.